### ISSUE TITLE
[Bug] Portfolio/Bench - Error: "Timestamp('2022-08-01 00:00:00')] not in index" [Markets for tickers in holdings may not have the same holiday calendar as the Benchmark]

### ISSUE THREAD

**Issue by deeleeramone** [OpenBB-finance/OpenBBTerminal#2362]
```
2022 Aug 18, 20:16 /portfolio/ $ bench SPDR Gold Shares (GLD)
Error: "[Timestamp('2021-12-27 00:00:00'), Timestamp('2021-12-28 00:00:00'), Timestamp('2022-01-03 00:00:00'), Timestamp('2022-05-23 00:00:00'), Timestamp('2022-07-01 00:00:00'), Timestamp('2022-08-01 00:00:00')] not in index"
```

These are user reports of the `bench` function not working when the user loads a custom portfolio file. I have recreated the scenarios using a sample file provided. 

The date column insists on populating with a time at midnight, despite those fields not being included in the CSV file. This may be a source of interference which is not obvious when using `example.csv`. 

<img width="1323" alt="Screen Shot 2022-08-18 at 5 29 48 PM" src="https://user-images.githubusercontent.com/85772166/185517350-dc92f75d-8104-4cc1-85b4-26d4fe02bab3.png">

To see if it was columns in the wrong order, or Sector/Industry being wrongly named, I ordered them exactly as they are in the `example.csv` file

<img width="1273" alt="Screen Shot 2022-08-18 at 5 45 02 PM" src="https://user-images.githubusercontent.com/85772166/185518660-b4f0e735-f1a1-497d-8ecf-f824e2fb2497.png">

Same result as before, and here, I removed the values from there and it self-populated. [this worked well, btw!]

<img width="1321" alt="Screen Shot 2022-08-18 at 5 40 09 PM" src="https://user-images.githubusercontent.com/85772166/185518331-aaaf0989-3824-420e-bd12-2c0bd859155e.png">

As you can see, the Benchmark is selected.

<img width="1326" alt="Screen Shot 2022-08-18 at 5 34 40 PM" src="https://user-images.githubusercontent.com/85772166/185517757-f6612e24-daf9-485a-b948-66768527679f.png">

The functions do work, despite this error. 

![Screen Shot 2022-08-18 at 5 33 56 PM](https://user-images.githubusercontent.com/85772166/185517697-a3f1b97d-bd7c-488e-bcc0-e3f201eb91d1.png)

![Screen Shot 2022-08-18 at 5 50 51 PM](https://user-images.githubusercontent.com/85772166/185519122-5fffaa7a-c9b1-466c-ba61-d6b658d74771.png)

<img width="570" alt="Screen Shot 2022-08-18 at 5 54 30 PM" src="https://user-images.githubusercontent.com/85772166/185519478-6f4caa67-a311-4646-83e1-538f695ece08.png">

The `summary` function does fail:

<img width="654" alt="Screen Shot 2022-08-18 at 5 52 28 PM" src="https://user-images.githubusercontent.com/85772166/185519277-a1774ef9-f750-4117-9738-04b3bfbdbad5.png">

In conclusion, I believe the source of this error is the market holiday calendars are different between the portfolio holdings (Canada) and the benchmark (USA). 

![Screen Shot 2022-08-18 at 5 57 05 PM](https://user-images.githubusercontent.com/85772166/185519749-1c173e5d-1b58-46a9-adfb-70464bebd377.png)

**Comment by deeleeramone** [comment]
related to this issue, the command `var`is also returning an error:
```
2022 Aug 19, 11:51 /portfolio/ $ var
Error: "None of [Index(['adjclose'], dtype='object', name='Date')] are in the "
```

**Comment by DidierRLopes** [comment]
Without the file being attached or sent offline, there's no way we can reproduce this easily. @deeleeramone

**Comment by deeleeramone** [comment]
Just sent you the file on Slack.

**Comment by deeleeramone** [comment]
```
Date,Ticker,Type,Sector,Industry,Country,Price,Quantity,Fees,Premium,Investment,Side,Currency,Region
2021-10-27,PHYS.TO,Stock,,,Canada,17.65,3000,200,0,53150,Buy,CAD,North America
2021-10-29,WPM.TO,Stock,,,Canada,51.23,300,200,0,15544,Buy,CAD,North America
2021-10-29,NGT.TO,Stock,,,Canada,71.05,200,200,0,14385,Buy,CAD,North America
2021-10-29,K.TO,Stock,,,Canada,7.93,1900,200,0,15232.5,Buy,CAD,North America
2021-11-04,PHYS.TO,Stock,,,Canada,17.57,1400,200,0,24773,Buy,CAD,North America
2021-11-04,ABX.TO,Stock,,,Canada,23.46,1000,200,0,23635,Buy,CAD,North America
2021-11-08,BTO.TO,Stock,,,Canada,5.42,1500,200,0,8305,Buy,CAD,North America
2021-11-15,FVI.TO,Stock,,,Canada,4.84,2000,200,0,9855,Buy,CAD,North America
2021-12-30,EDV.TO,Stock,,,Canada,27.91,550,200,0,15526.5,Buy,CAD,North America
2022-01-19,PHYS.TO,Stock,,,Canada,18.13,2800,200,0,50964,Buy,CAD,North America
2022-02-15,AEM.TO,Stock,,,Canada,65.91,350,200,0,23068.5,Buy,CAD,North America
2022-03-08,K.TO,Stock,,,Canada,7.23,1900,200,0,13912,Buy,CAD,North America
2022-03-24,PSLV.TO,Stock,,,Canada,11.34,3000,200,0,34220,Buy,CAD,North America
2022-03-24,PAAS.TO,Stock,,,Canada,36,600,200,0,21800,Buy,CAD,North America
2022-03-24,ABX.TO,Stock,,,Canada,31.2,1000,200,0,31000,Sell,CAD,North America
2022-03-24,NGT.TO,Stock,,,Canada,99.08,200,200,0,22786.24,Sell,CAD,North America
2022-03-25,CG.TO,Stock,,,Canada,12.31,1750,200,0,21742.5,Buy,CAD,North America
2022-03-25,BTO.TO,Stock,,,Canada,5.82,3400,200,0,19988,Buy,CAD,North America
2022-03-30,CG.TO,Stock,,,Canada,12.28,1600,200,0,20099.9,Buy,CAD,North America
2022-04-11,PSLV.TO,Stock,,,Canada,11.13,1800,200,0,20234,Buy,CAD,North America
2022-04-11,MOZ.TO,Stock,,,Canada,2.81,7000,200,0,19891,Buy,CAD,North America
2022-05-11,CEF.TO,Stock,,,Canada,22.82,450,200,0,10444.5,Buy,CAD,North America
```

**Comment by montezdesousa** [comment]
Hey Mr. @deeleeramone , you were not forgotten. 

I think the issue is now solved by this https://github.com/OpenBB-finance/OpenBBTerminal/pull/2143. I was able to load your file without problems.
